src/komponenter/kalkulator/DatointervallInput/DatointervallInput.tsx

```diff
@@ -85,6 +86,13 @@ const DatoIntervallInput: FunctionComponent<Props> = (props) => {
                 aria-label="Slett periode"
                 onClick={props.slettPeriode}
             />
+            <Element
+                className="datointervall-input__feilmelding"
+                aria-live="assertive"
+                aria-label={'feilmelding'}
```

Comment:
Hvis man legger på en `aria-label`, vil denne labelen bli lest opp i stedet for selve innholdet. Hvis vi vil at "feilmelding" skal leses opp for skjermlesere _i tillegg til_ selve feilmeldingen, så kan man legge på f.eks. en tom `span` med `aria-label` på seg

Reply:
på min skjermleser blir begge to lest opp. aria-label skriver ikke over teksten, men sier i tillegg på voiceover i chrome